report_ai/index.html

```diff
@@ -35,7 +35,7 @@
     <div style = "width:75%"><h1>Report.AI</h1></div>
     <div style = "display:flex; flex-direction:column; margin-left:7%">
       <div style="height:70%;"><button title="Reload Current-tab" id = "refreshBtn">Reload</button></div>
-      <div class = "connectionSignal"> </div>
+      <div class = "connectionSignal" id="connectionSignal"> </div>
     </div>

```

Comment:
Can we add a title="waiting for connection"
and in the script, 
if server alive, set title = "connected to server"
else title = "server unavailable"


Everything else looks good :fire:

Reply:
yeah, i will make the changes. Thanks for the review

Reply:
Enhance connection signal element with text and icon integration. Review requested.